<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>finestraPrincipal</class>
 <widget class="QMainWindow" name="finestraPrincipal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>526</width>
    <height>460</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LaTeX2GIFT v1.1</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Ficheros</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QListWidget" name="llistaFitxersTex"/>
          </item>
          <item>
           <widget class="QListWidget" name="llistaFitxersGIFT"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QPushButton" name="botoAfegir">
            <property name="text">
             <string>Agregar Ficheros...</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="botoLlevarFitxer">
            <property name="text">
             <string>Quitar Ficheros</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Mensajes</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QTextBrowser" name="logConversio"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QProgressBar" name="barraProgress">
      <property name="value">
       <number>0</number>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="botoConvert">
        <property name="text">
         <string>Convertir</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="botoSortir">
        <property name="text">
         <string>Salir</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>526</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>botoSortir</sender>
   <signal>clicked()</signal>
   <receiver>finestraPrincipal</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>518</x>
     <y>410</y>
    </hint>
    <hint type="destinationlabel">
     <x>510</x>
     <y>358</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
